proxy-robots-module/src/main/java/orwell/proxy/IMessageFramework.java

```diff
@@ -0,0 +1,26 @@
+package orwell.proxy;
+
+/**
+ * Created by parapampa on 03/05/15.
+ */
+public interface IMessageFramework {
```

Comment:
We may have talked about this long ago. I am not a big fan of objects named framework as to me it does not mean anything and might better apply (if one is so inclined) to something bigger than a single class. Maybe MessageHub might be better (this name has flaws so it is more to start the discussion).